src/system-health/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --cov=health_checker --cov-report html --cov-report term --cov-report xml
```

Comment:
Does this also report coverage for `healthd`? Maybe you need to add an extra `--cov=scripts`?

Reply:
The current unit test doesn't test healthd, because healthd has a dead loop and most of the logic is in health_checker.

Reply:
OK. If healthd is ever modified such that tests can be added, we'll add it to the coverage at that time.